python/ray/serve/schema.py

```diff
@@ -197,38 +231,9 @@ def num_replicas_and_autoscaling_config_mutually_exclusive(cls, values):
 
         return values
 
-    @validator("route_prefix")
-    def route_prefix_format(cls, v):
-        """
-        The route_prefix
-        1. must start with a / character
-        2. must not end with a / character (unless the entire prefix is just /)
-        3. cannot contain wildcards (must not have "{" or "}")
-        """
-
-        # route_prefix of None means the deployment is not exposed
-        # over HTTP.
-        if v is None or v == DEFAULT.VALUE:
-            return v
-
-        if len(v) < 1 or v[0] != "/":
-            raise ValueError(
-                f'Got "{v}" for route_prefix. Route prefix ' 'must start with "/".'
-            )
-        if v[-1] == "/" and len(v) > 1:
-            raise ValueError(
-                f'Got "{v}" for route_prefix. Route prefix '
-                'cannot end with "/" unless the '
-                'entire prefix is just "/".'
-            )
-        if "{" in v or "}" in v:
-            raise ValueError(
-                f'Got "{v}" for route_prefix. Route prefix '
-                "cannot contain wildcards, so it cannot "
-                'contain "{" or "}".'
-            )
-
-        return v
+    _route_prefix_format = validator("route_prefix", allow_reuse=True)(
```

Comment:
```suggestion
    deployment_schema_route_prefix_format = validator("route_prefix", allow_reuse=True)(
```

The name change helps a bit with readability. Right now, the reader has to consider whether overwriting `_route_prefix_format` with a decorate version of itself affects other parts of the code that use `_route_prefix_format`.